src/nvim/spell.c

```diff
@@ -67,37 +67,23 @@
 // for offsetof()
 #include <stddef.h>
 
-#include "nvim/ascii.h"
 #include "nvim/buffer.h"
 #include "nvim/change.h"
 #include "nvim/charset.h"
 #include "nvim/cursor.h"
 #include "nvim/drawscreen.h"
-#include "nvim/edit.h"
 #include "nvim/ex_cmds.h"
 #include "nvim/ex_docmd.h"
-#include "nvim/func_attr.h"
-#include "nvim/garray.h"
-#include "nvim/hashtab.h"
-#include "nvim/input.h"
```

Comment:
Just checking: does the removal of these headers depend on symbols included transitively? I believe that would be contrary to the IWYU goal https://github.com/neovim/neovim/issues/549 .

Reply:
Wasn't aware of the IWYU goal. I've ran it through include-what-you-use